daml-lf/transaction/src/main/scala/com/daml/lf/Error.scala

```diff
@@ -146,6 +146,8 @@ object Error {
 
   final case class ContractIdInContractKey(key: Value) extends Error
 
+  final case class ValueNesting(location: String, limit: Int) extends Error
```

Comment:
Adding the `location` param from the `Dev` wrapping case class - this will likely be useful for tooling and error reporting?

Reply:
cf above.